src/Components/Draw/Draw.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="container">
+		<ul>
+			<li
+				:class="{active: isActive('text')}"
+				@click.prevent="setActive('text')">
+				<img :src="texticon" alt="Text">
+				Text
+			</li>
+			<li
+				:class="{active: isActive('draw')}"
+				@click.prevent="setActive('draw')">
+				<img :src="drawnIcon" alt="draw">
```

Comment:
I believe is a good aproach implement translation on the alt text.

Reply:
This PR is in Draft, it is incomplete.